pkg/controllers/jobset_controller.go

```diff
@@ -46,6 +46,16 @@ import (
 	"sigs.k8s.io/jobset/pkg/util/placement"
 )
 
+const (
+	legacyJobNameLabel       = "job-name"
```

Comment:
These labels should be defined in the Jobset api. I think they already exist there.

Reply:
sgtm, updated.